azure-pipelines.yml

```diff
@@ -130,6 +130,17 @@ jobs:
       HADOOP_HOME: $(Build.BinariesDirectory)\hadoop
       DOTNET_WORKER_DIR: $(Build.ArtifactStagingDirectory)\Microsoft.Spark.Worker\netcoreapp2.1\win-x64
```

Comment:
Should we add a table in README.md regarding supported versions? Since 2.4.2 is not supported, we should mention that we don't and add the reason (link to that thread).